dbos/_debug.py

```diff
@@ -0,0 +1,40 @@
+import re
+import runpy
+from typing import Union
+
+from dbos import DBOS
+
+
+class PythonModule:
+    def __init__(self, module_name: str):
+        self.module_name = module_name
+
+
+def debug_workflow(workflow_id: str, entrypoint: Union[str, PythonModule]) -> None:
+    if isinstance(entrypoint, str):
+        runpy.run_path("app/main.py")
```

Comment:
Should this by `runpy.run_path(entrypoint)`, not the hard coded "app/main.py"?

Reply:
Uhh, yes. Woops.